marimo/_server/ai/providers.py

```diff
@@ -402,18 +431,34 @@ def stream_completion(
             ),
             system=system_prompt,
             stream=True,
-            temperature=0,
+            temperature=1,  # Reasoning requires temperature > 0
```

Comment:
is this going to give different results for non-reasoning? happy to change temperature. im not sure if 0 was the most optimal anyways, but not sure if 1 is either.